### ISSUE TITLE
Error message flashes before flow starts

### ISSUE THREAD

**Issue by st3fan** [mozilla/persona#1475]
When the BrowserID code is loaded in a UIWebView it initially show a message in red text about some error that has occurred. After a little while that message goes away and it will continue with the normal flow.

On mobile networks, where loading times are much slower, the message sometimes stays on screen for a few seconds.

I think this message should be initially hidden and only shown when actually needed.

This might be Pancake specific.

**Comment by jrgm** [comment]
I noticed this on an iPhone 4s today, with Safari + dev.diresworb.org, and my network was sluggish. The message I got 
could be seen on either the /signin or /signup page when rapidly clicking from one to the other. The message was about 
cookies disabled. On a late Friday afternoon whim, I used ipfw to induce some latency on my osx laptop, and could reproduce it on FF12. 

@st3fan did you ever click the 'more info' on those error messages. Did it mention something about cookies?
